packages/koa-performance/src/middleware.ts

```diff
@@ -47,7 +47,8 @@ export function clientPerformanceMetrics({
   statsdHost,
   statsdPort,
   prefix,
-  development,
+  // eslint-disable-next-line no-process-env
+  development = process.env.NODE_ENV === 'development',
```

Comment:
if we are changing API, I kinda want to change this option from `development` to `disabledLogging`

Reply:
I only made it optional because it is always passed the same value and was annoying in the examples. If rather just remove the API change than make it actually breaking

Reply:
I only made it optional because it is always passed the same value and was annoying in the examples. If rather just remove the API change than make it actually breaking

Reply:
I'll keep this for now since it's just making an existing API optional, we can follow up in another PR to make breaking changes :)